src/lib/tools/bookingAssistantTool.ts

```
@@ -0,0 +1,378 @@
+import { z } from "zod";
+import connectDB from '@/lib/db';
+import Department from '@/lib/models/departmentSchema';
+
+interface Service {
+  _id: string;
+  name: string;
+  description?: string;
+  isActive: boolean;
+}
+
+interface Department {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Department'. Consider to delete it or rename it.

'Department' is defined here:


(lint/suspicious/noRedeclare)